ifbcat_api/model/tool/tool.py

```diff
@@ -142,6 +145,8 @@ def update_information_from_json(self, tool: dict):
         # software_version = tool['version']
         # downloads = tool['download']
         self.tool_license = tool['license']
+        for doc in tool['documentation']:
+            self.documentation = doc['url']
```

Comment:
We indeed can have multiple documentation (I added 2 for https://bio.tools/api/edam-browser). I think you should be more selective, and chose between the different type of documentation the one you would prefer to have. I let you find the different type of doc, and how to chose to most relevant. I think you should propose an ordering of documentation type by relevant-ness.

Reply:
I also notice that we've type field on documentation. Should we only take documentation url where the type is different to _Other_ ?

Reply:
For the edam-browser indeed we should something else than other, but there are many type of documentation, I recommend you to find the list of type of documentation so we can chose what type we want to use.

Reply:
ok I'm on it 👍🏾

Reply:
Done in metting in https://github.com/IFB-ElixirFr/ifbcat/issues/181#issuecomment-1017261102